po/README

```diff
@@ -1,3 +1,5 @@
+Portuguese
```

Comment:
why?

Reply:
Ohhh, sorry.
Meant to stay. Will gitignore